app-backend/batch/src/main/scala/com/azavea/rf/batch/util/S3.scala

```diff
@@ -109,9 +109,11 @@ object S3 {
 
   /** Set credentials in case Hadoop configuration files don't specify S3 credentials. */
   def setCredentials(conf: Configuration, credentialsProviderChain: AWSCredentialsProvider = new DefaultAWSCredentialsProviderChain): Configuration = {
-    conf.set("fs.s3.impl", classOf[org.apache.hadoop.fs.s3native.NativeS3FileSystem].getName)
-    conf.set("fs.s3.awsAccessKeyId", credentialsProviderChain.getCredentials.getAWSAccessKeyId)
-    conf.set("fs.s3.awsSecretAccessKey", credentialsProviderChain.getCredentials.getAWSSecretKey)
+    if(conf.isKeyUnset("fs.AbstractFileSystem.s3a.impl")) {
```

Comment:
I just noticed that this flag is setuped on EMR configuration. (full dump of all Configuration key -> values: https://gist.github.com/pomadchin/1d0ddc7cb4b3f3931786f3a9abd14baf)

The logic here that if this `fs.AbstractFileSystem.s3a.impl` setting is not setuped than force s3n usage, otherwise use default configuration settings provided by server. 

In fact this `setCredentials` function is required only for development purposes as on well configured systems it should be already well.

Do you think it makes more sense to add a development falg rahter than using that trick with identifying EMR?